### PR TITLE
PAN-OS - panorama-show-location-ip - Get the location of an IP address.

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/24651

## Minimum version of Demisto
- [x] 3.5.0

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Tests - tested both on fw and panorama
- [x] Documentation

### CODE REVIEW

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py [hunk 1]
```diff
@@ -5310,6 +5310,57 @@ def panorama_device_reboot_command():
...
+
+
+def panorama_show_location_ip_command():
+    """
+    Check location of a specified ip address
+    """
+    ip_address = demisto.args().get('ip_address')
+    result = panorama_show_location_ip(ip_address)
+
+    if result['response']['@status'] != 'success':
```
Comment (yuvalbenshalom) [inline Packs/PAN-OS/Integrations/Panorama/Panorama.py:5336]: maybe cache that @status is there before?

**Comment by bakatzir** [comment]
fails on: `PAN-OS Create Or Edit Rule Test` which fails on nightly.

**Comment by amshamah419** [comment]
@bakatzir Docs look good

**Comment by bakatzir** [comment]
force merging due to the non related failure.

**Pull request merged**